packages/edit-site/src/components/page-patterns/grid.js

```diff
@@ -1,26 +1,113 @@
 /**
  * WordPress dependencies
  */
-import { __experimentalText as Text } from '@wordpress/components';
-import { useRef } from '@wordpress/element';
-import { __, sprintf } from '@wordpress/i18n';
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	Button,
+} from '@wordpress/components';
+import { useRef, useState, useMemo } from '@wordpress/element';
+import { __, _x, sprintf } from '@wordpress/i18n';
+import { useAsyncList } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
 import GridItem from './grid-item';
 
-const PAGE_SIZE = 100;
+const PAGE_SIZE = 20;
+
+function Pagination( { currentPage, numPages, changePage, totalItems } ) {
+	return (
+		<HStack
+			expanded={ false }
+			spacing={ 3 }
+			className="edit-site-patterns__grid-pagination"
+		>
+			<Text variant="muted">
+				{
+					// translators: %s: Total number of patterns.
+					sprintf( __( '%s items' ), totalItems )
```

Comment:
I think we should use `_n` method here:

```js
_n( '%d item', '%d items', totalItems )
```

Reply:
yep, done.

Reply:
@glendaviesnz, I think you forgot the singular translation string for `_n`.

Reply:
Have added, but on thinking about it this string won't display unless there are 20+ items so I wonder if we do need `_n` 🤔  - I guess it won't hurt just in case that paging component ends up being used somewhere that isn't the case.